api/graphql/reservations/reservation_serializers.py

```diff
@@ -295,6 +296,15 @@ def check_reservation_start_time(self, begin, scheduler):
                 f"Reservation start time does not match the allowed interval of {interval_minutes} minutes."
             )
 
+    def check_metadata_fields(self, data, reservation_unit) -> None:
+        metadata_set = reservation_unit.metadata_set
+        required_fields = metadata_set.required_fields.all() if metadata_set else []
+        for required_field in required_fields:
+            if not data.get(required_field.field_name):
+                raise serializers.ValidationError(
+                    f"Value for required field {required_field} is missing."
```

Comment:
I assume the `ReservationMetadataField` `field_name` is not camel cased? If so this should  use graphene's `to_camel_case` function.

```
from graphene.utils.str_converters import to_camel_case
.....
f"Value for required field {to_camel_case(required_field)} is missing."
```

Reply:
Good point, also I forgot to use `required_field.field_name` instead of just `required_field`.